clang/test/dpct/syncthreads.cu

```diff
@@ -185,6 +188,9 @@ __global__ void test10(float *arg1_ptr, int arg2_scalar, int arg3_scalar) {
       }
       // CHECK: var3 = var3 / 2;
       // CHECK-NEXT: /*
+      // CHECK-NEXT: DPCT1115:{{[0-9]+}}: Call __syncthreads inside a conditional branch statement may lead to undefined behavior
```

Comment:
SYCL group functions and algorithms must be encountered in converged control flow. You should check this condition holds.

Reply:
fixed